module/dkg/broker_test.go

```diff
@@ -259,6 +259,7 @@ func TestBroadcastMessage(t *testing.T) {
 	sender.dkgContractClient = contractClient
 
 	sender.Broadcast(msgb)
+	<-sender.unit.Done()
```

Comment:
```suggestion
	unittest.AssertClosesBefore(sender.unit.Done(), time.Second)
```
We have a utility to wrap waiting for a channel to exit, to prevent a bug blocking the tests and making such bugs more obvious.